x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/utils.ts

```diff
@@ -52,6 +52,16 @@ export const createBulkErrorObject = ({
   };
 };
 
+export interface ImportRuleResponses {
```

Comment:
Since this is a single object I would consider dropping the 's' on this interface to go from `ImportRuleResponses` to `ImportRuleResponse` as it's only a single object.